fineract-provider/src/main/resources/application.properties

```diff
@@ -201,3 +201,9 @@ fineract.integrations.metropol.crb.rest.clientVerify=${METROPOL_CRB_REST_POST_CL
 fineract.integrations.metropol.crb.rest.publicKey=${METROPOL_CRB_REST_PUBLIC_KEY:XXXXXX}
 fineract.integrations.metropol.crb.rest.privateKey=${METROPOL_CRB_REST_PRIVATE_KEY:XXXXXX}
 fineract.integrations.metropol.crb.rest.clientVerifyCreditInfoEnhanced=${METROPOL_CRB_REST_POST_CLIENT_VERIFY_CREDIT_INFO_ENHANCED_URL:https://api.metropol.co.ke:5555/v2_1/report/credit_info_enhanced}
+
+#Disbursement integration
+fineract.integrations.inkomoko.rest.username=${INKOMOKO_REST_USERNAME:XXXXXX}
+fineract.integrations.inkomoko.rest.password=${INKOMOKO_REST_PASSWORD:XXXXXX}
+fineract.integrations.inkomoko.rest.authenticationUrl=${INKOMOKO_REST_AUTHENTICATION_URL:XXXXXX}
```

Comment:
@tabrez-fiter  For Properties that are not sensitive please have them here. Instead of placeholder. This will reduce on the config attribute in docker-compose file of env variable